config/schedule.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 env :PATH, ENV['PATH']
 
-# Every minute, check for departure inaccuracies
-every 60.seconds do
+# Every minute after 5am, check for departure inaccuracies
+# raw cron asterisk order: minute, hour, day of month, month, day of week
+every '*/1 6-23 * * *' do
```

Comment:
Alright, I can change it.

Since 0 is midnight, it starts running after 6am. I'll fix that.